run_integration_test.go

```diff
@@ -184,7 +186,7 @@ func buildCustomRestic(t *testing.T) {
 
 	fmt.Println(string(out))
 
-	checkout := exec.Command("git", "checkout", "tar")
+	checkout := exec.Command("git", "checkout", resticVersion)
 	checkout.Dir = "tmp/restic"
 	checkoutOut, err := checkout.CombinedOutput()
```

Comment:
The output is captured into memory only to be written to the output again (mixing stderr and stdout). Instead `cmd.Stdout` and `cmd.Stderr` can be set to ones of the calling application. The output will then be written directly to the console and hardly any additional memory will be needed.